src/entity/mesh.hpp

```diff
@@ -333,11 +349,22 @@ ambient_occlusion_mesher(T voxel_object) {
                             current_vertex_index++;
                         }
                     }
-
+                    /*clange-format off*/
+                    /* Figure 1 a
+                      0 ------- 1  Wow did you know a square a actually two
+                      |       / |  triangles?! This is high level geometry
+                      |  1  /   |  that many people don't know. Now add the
+                      |   /  2  |  vertices in a counter-clock wise direction
+                      | /       |  so that the opengl thinks the normal is
+                      2 ------- 3  pointing toward the viewer.
+                    */
+                    /*clang-format on*/
```

Comment:
```suggestion
                    /* clang-format off */

                    /* Figure 1 a
                      0 ------- 1  Wow did you know a square a actually two
                      |       / |  triangles?! This is high level geometry
                      |  1  /   |  that many people don't know. Now add the
                      |   /  2  |  vertices in a counter-clock wise direction
                      | /       |  so that the opengl thinks the normal is
                      2 ------- 3  pointing toward the viewer.
                    */

                    /* clang-format on */
```